certora/README.md

```diff
@@ -0,0 +1,48 @@
+# Running the certora verification tool
+
+These instructions detail the process for running CVT on the OpenZeppelin (Wizard/Governor) contracts.
+
+Documentation for CVT and the specification language are available
+[here](https://certora.atlassian.net/wiki/spaces/CPD/overview)
+
+## Running the verification
+
+The scripts in the `certora/scripts` directory are used to submit verification
+jobs to the Certora verification service. These scripts should be run from the
+root directory; for example by running
+
+```
+sh certora/scripts/WizardFirstPriority.sh <meaningful comment>
+```
+
+After the job is complete, the results will be available on
+[the Certora portal](https://vaas-stg.certora.com/).
```

Comment:
I'm told that this is the non-staging site and it's the one I get by default when running the CLI.

```suggestion
[the Certora portal](https://portal.certora.com/).
```

Reply:
The verification uses some features that haven't been merged into our master yet; the scripts pass the --staging option, which causes the reports to get output here.  I'll file a new PR updating the scripts and README when those features land.